src/features/folder_creation/manual_problem_folder.ts

```diff
@@ -39,6 +39,8 @@ export const manualProblemFolderCreation = async (
     catch (err) {
         if (err.code === ErrorCodes.folderExists) {
             vscode.window.showErrorMessage("Problem folder already exists");
+        } else if(err.code ===ErrorCodes.noAccessPermission) {
+            vscode.window.showErrorMessage("No access permission.");
```

Comment:
The error message in the case of noAccessPermission in manual problem folder creation and problem folder creation says " no access permission " but it says "Please open a folder in your workspace" for contest folder creations in this PR. I think we could show the same error message ("Please open a folder in your workspace") for problem folders and contest folders